apps/concept-catalog/app/catalogs/[catalogId]/concepts/search-page-client.tsx

```diff
@@ -14,7 +14,13 @@ import {
   LinkButton,
   SearchField,
 } from '@catalog-frontend/ui';
-import { getTranslateText, capitalizeFirstLetter, localization as loc, localization, setClientConceptsPageSettings } from '@catalog-frontend/utils';
+import {
+  getTranslateText,
+  capitalizeFirstLetter,
+  localization as loc,
+  localization,
```

Comment:
Var vel ikke du som la det til nå, men er uansett lite vits å importere localization 2 ganger
```suggestion
  localization,
```